Description: Disable location panel, automatically enable all computers.
 As puavo uses own session system, these features are not required inside
 veyon, and are thus disabled / hidden.
Author: Tuomas Nurmi <dev@opinsys.fi>

--- veyon-4.7.4+repack1.orig/master/src/ComputerManager.cpp
+++ veyon-4.7.4+repack1/master/src/ComputerManager.cpp
@@ -435,10 +435,10 @@ ComputerList ComputerManager::selectedCo
 	{
 		QModelIndex entryIndex = model->index( i, 0, parent );
 
-		if( QVariantHelper<Qt::CheckState>::value( model->data( entryIndex, NetworkObjectModel::CheckStateRole ) ) == Qt::Unchecked )
+		/*if( QVariantHelper<Qt::CheckState>::value( model->data( entryIndex, NetworkObjectModel::CheckStateRole ) ) == Qt::Unchecked )
 		{
 			continue;
-		}
+		}*/
 
 		const auto objectType = NetworkObject::Type( model->data(entryIndex, NetworkObjectModel::TypeRole).toInt() );
 
--- veyon-4.7.4+repack1.orig/master/src/MainWindow.cpp
+++ veyon-4.7.4+repack1/master/src/MainWindow.cpp
@@ -108,12 +108,13 @@ MainWindow::MainWindow( VeyonMaster &mas
 	monitoringSplitter->setStretchFactor( monitoringSplitter->indexOf(slideshowSpotlightSplitter), 1 );
 	monitoringSplitter->setStretchFactor( monitoringSplitter->indexOf(ui->computerMonitoringWidget), 1 );
 
-	mainSplitter->addWidget( computerSelectPanel );
+	//mainSplitter->addWidget( computerSelectPanel );
 	mainSplitter->addWidget( screenshotManagementPanel );
 	mainSplitter->addWidget( monitoringSplitter );
 
 	mainSplitter->setStretchFactor( mainSplitter->indexOf(monitoringSplitter), 1 );
 
+       ui->computerSelectPanelButton->setVisible(false);
 
 	static const QMap<QWidget *, QAbstractButton *> panelButtons{
 		{ computerSelectPanel, ui->computerSelectPanelButton },
@@ -190,10 +191,10 @@ MainWindow::MainWindow( VeyonMaster &mas
 
 	ui->centralLayout->addWidget( mainSplitter );
 
-	if( VeyonCore::config().autoOpenComputerSelectPanel() )
+	/*if( VeyonCore::config().autoOpenComputerSelectPanel() )
 	{
 		ui->computerSelectPanelButton->setChecked( true );
-	}
+	}*/
 
 	// initialize search filter
 	ui->filterPoweredOnComputersButton->setChecked( m_master.userConfig().filterPoweredOnComputers() );
